b.py

```diff
@@ -6,9 +6,7 @@ def plus(a,b):
     if plus(3,4) == 3+4:
         print("통과!!!!")
 
-
-a = 100
-print(a+200)
-print(300)
-print(1903912389)
-print(290923109)
\ No newline at end of file
+n = int(input())
+sum = 0
```

Comment:
야이 병신아 제발 대가리를 달았으면 써먹여야지 하...
```suggestion
n = int(input())
sum=n*(n+1)/2
```
알아먹자